### PR TITLE
markdown-toc 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by PythonCoderAS**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/markdown-toc.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  desc "Generate a markdown TOC (table of contents) with Remarkable"
+  homepage "https://github.com/jonschlinkert/markdown-toc"
+  url "https://registry.npmjs.org/markdown-toc/-/markdown-toc-1.2.0.tgz"
+  sha256 "4a5bf3efafb21217889ab240caacd795a1101bfbe07cd8abb228cc44937acd9c"
+  license "MIT"
+
+  livecheck do
+    url :stable
+  end
+
```
Comment (chenrui333) [inline Formula/markdown-toc.rb:13]:
```suggestion
```

File: Formula/markdown-toc.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_equal "- [One](#one)\n- [Two](#two)",
```
Comment (PythonCoderAS) [inline Formula/markdown-toc.rb:22]:
```suggestion
    assert_equal "- [One](#one)\n- [Two](#two)",
```

File: Formula/markdown-toc.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      shell_output("bash -c \"#{bin}/markdown-toc - <<< $'# One\\n\\n# Two'\"").strip
```
Comment (carlocab) [inline Formula/markdown-toc.rb:23]: Doesn't `pipe_output` work here?

Comment (PythonCoderAS) [reply Formula/markdown-toc.rb:23]: I'm not really sure how the tests work, I based my test by looking at another formula

**Pull request closed without merging**